### PR TITLE
Update BBE workflow to generate 1.1.4 BBE artifacts

### PR DESCRIPTION
**Pull request by shafreenAnfar**

### CODE REVIEW

**Comment by NipunaMarcus** [comment]
@shafreenAnfar we might need to update go version to the latest as i am using strings.ReplaceAll which was available after  v1.12 of golang..

File: .github/workflows/ballerina-by-example.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ jobs:
       uses: actions/checkout@v2
 
     - name: Generate BBEs
-      run: ./ballerinaByExample/build-bbe.sh v1-2 bbes v1.2.2 true
+      run: ./ballerinaByExample/build-bbe.sh v1-1 bbes v1.1.4 true false
```
Comment (NipunaMarcus) [inline .github/workflows/ballerina-by-example.yml:20]: Here the site version should be 1.1 isn't it? As we now changed the way we represent site versions

Comment (shafreenAnfar) [reply .github/workflows/ballerina-by-example.yml:20]: having dot isn't an issue ?

Comment (NipunaMarcus) [reply .github/workflows/ballerina-by-example.yml]: I checked already changed .me files available in 1.1/learn ... They seem to work

**Pull request merged**